psydac/api/discretization.py

```diff
@@ -81,18 +81,42 @@ def change_dtype(V, dtype):
 
     return V
 
-#==============================================================================
-def discretize_derham(derham, domain_h, *args, **kwargs):
+#==============================================================================           
+def discretize_derham(derham, domain_h, get_vec = False, *args, **kwargs):
+    """
+    Create a discrete De Rham sequence by creating the spaces and then initiating DiscreteDerham object.
+
+    Parameters
+    ----------
+
+    derham : sympde.topology.space.Derham
+        The symbolic Derham sequence
+
+    domain_h   : Geometry
+        Discrete domain where the spaces will be discretized
+        
+    get_vec : Bool
```

Comment:
to be consistent with the notation used elsewhere, I would call this argument `get_H1vec_space`